docs/useful-tips/transform-function-and-its-parameters.md

```diff
@@ -153,3 +155,7 @@ const result2 = image.transform(rotationMatrix, {
 This is particularly useful for rotations, where corners of the image may extend beyond the original boundaries.
 
 ![fullImage test](./images/transform-parameters/fullImageTest.svg)
+
+:::warning
+If `fullImage` is defined together with `width` and `height`, the algorithm will use dimensions calculated for `fullImage`.
```

Comment:
I'd phrase differently and say that the `width` and `height` parameters are ignored in that case.